src/test/java/org/folio/innreach/domain/listener/KafkaCirculationEventListenerApiTest.java

```diff
@@ -276,6 +283,36 @@ void shouldSkipIfTransactionNotFoundByRequestId() {
     verify(innReachExternalService, times(0)).postInnReachApi(any(), any(), any());
   }
 
+  @Test
+  @Sql(scripts = {
+    "classpath:db/central-server/pre-populate-central-server.sql",
+    "classpath:db/inn-reach-transaction/pre-populate-another-inn-reach-transaction.sql"
+  })
+  void shouldUpdateTransactionWhenCancelRequest() {
+    var folioRequestId = UUID.fromString("26278b3a-de32-4deb-b81b-896637b3dbeb");
+    var transactionId = UUID.fromString("2d219267-4e51-4843-b2c6-d9c44d313739");
+    var instanceId =  UUID.fromString("18fedcfa-81d4-4c29-a7ee-2540f57cca71");
```

Comment:
typically when we refer to pre populated data of test scripts we create constants for this
https://github.com/folio-org/mod-inn-reach/blob/1464b3971f01fc187cc653f01bb63d8369f2b8cd/src/test/java/org/folio/innreach/domain/listener/KafkaCirculationEventListenerApiTest.java#L74-L77

Reply:
Fixed